src/main/java/com/techzen/techlearn/service/impl/ChapterServiceImpl.java

```diff
@@ -73,7 +73,19 @@ public void deleteChapter(Long id) {
 
     @Override
     public PageResponse<?> getAllChapters(int page, int pageSize) {
```

Comment:
không hiểu cái chổ sắp xếp này lắm, anh lấy tất cả xong sắp xếp theo thứ tự theo chapterOrder thôi
vd: @Override
    public PageResponse<?> getAllLesson(int page, int pageSize, Long idChapter) {
        Pageable pageable = PageRequest.of(page > 0 ? page - 1 : 0, pageSize,
                Sort.by("lessonOrder"));
        Page<LessonEntity> course = lessonRepository.findAllByChapterId(idChapter, pageable);
        List<LessonResponseDTO> list = course.map(lessonMapper::toLessonResponseDTO)
                .stream().collect(Collectors.toList());
        return PageResponse.builder()
                .page(page)
                .pageSize(pageSize)
                .totalPage(course.getTotalPages())
                .items(list)
                .build();
    }